AFQ/api.py

```diff
@@ -945,6 +1070,9 @@ def _get_best_scalar(self):
         return self.scalars[0]
 
     def get_reg_template(self):
+        if isinstance(self.reg_template, nib.Nifti1Image):
```

Comment:
@36000 : what do you think? Does this change make sense? Or should this go somewhere else?

Reply:
@36000 : do you have any thoughts here? I would like to merge this PR, but need some input from you for this particular line.